flatliners/stddevcluster.py

```diff
@@ -34,9 +35,32 @@ def calculate_mean(values, previous = None):
         if previous:
             count = len(values) + previous['count']
             v_sum = previous['count'] * previous['mean']
+
         else:
             count = len(values)
             v_sum = 0
 
         v_sum += sum(int(x[1]) for x in values)
         return {'count': count, 'mean': v_sum / count}
+
+    @staticmethod
+    def calculate_stdv(values, previous = None):
+        if previous:
+
+            total = previous['total'] + int(values[0][0]) # I'm not 100% sure if this is the correct way to organize the data.
```

Comment:
I'm a little unclear as to what is being passed as the `values` parameter. From inspection it looks like a list of multiple samples, where each sample is a list, with position [0] being some integer that represents the resource type (?) and position [1]  representing the actual value?

Reply:
It's the [metric_values](https://github.com/AICoE/prometheus-flatliner/blob/master/flatliners/baseflatliner.py#L33)
That's the array of `[timestamp, value]` pairs.

Actually, we could create another metrics processor that flattens these arrays. So you dont have to this in every processor.